dataland-api-key-manager/src/main/kotlin/org/dataland/datalandapikeymanager/services/ApiKeyManager.kt

```diff
@@ -25,8 +26,12 @@ class ApiKeyManager(
 
     companion object {
         private const val milliSecondsInADay = 86400000
+        private const val defaultMaxDaysOfValidityForApiKey = 365
```

Comment:
Bei `milliSecondsInADay` fände ich das Produkt ausgeschrieben ein bisschen nachvollziehbarer.

Reply:
Die Zahl steckt jetzt in einer constant im Constants.ts file, und dort ist das Produkt auch ausgeschrieben.

Reply:
Äh ne

Reply:
Sorry habe es mit Frontend verwechselt.

Also:   Habe das Produkt hier ausgeschrieben.  Check es gerne ab.